content/blog/2022/czi-global-communities-proposal/index.md

```diff
@@ -0,0 +1,62 @@
+
+{{< cta cta_text="Go to Zenodo record" cta_link="https://zenodo.org/record/7025288" cta_new_tab="true" >}}
+
+Read on for a quick overview of the proposal.
+
+## Collaborators
+
+This grant is a collaboration between several leading organizations in open infrastructure, community, and global leadership:
+
+- [2i2c](https://2i2c.org)
+- [The Carpentries](https://carpentries.org/about/)
+- [Center for Scientific Collaboration and Community Engagement](https://www.cscce.org/)
+- [Invest in Open Infrastructure](https://investinopen.org/)
+- [MetaDocencia](https://www.metadocencia.org/)
+- [Open Life Science](https://openlifesci.org/)
+
+## Problem statement
+
+Cloud infrastructure is a powerful way to broaden access to workflows and infrastructure across the globe.
+However, it is also inaccessible to many for a variety of reasons:
+
+- There is a large, diverse, and messy ecosystem of open source tools to facilitate cloud infrastructure.
+- Most communities don't already have skills in utilizing cloud workflows.
+- Running infrastructure in the cloud takes dedicated time and expertise that many communities lack.
+- Many communities do not have organized communities of practice around cloud infrastructure.
+
+These issues are true for most scientific communities, but they are exacerbated in countries that are historically under-represented in the global scientific community.
+
+## Our proposed work
+
+For this reason, our goal in this grant is to **provide human and technical services to facilitate learning and knowledge transfer of cloud workflows for communities in Latin America and Africa**.
+
+It defines four major areas of collaboration:
+
+- **Cloud infrastructure management** - to facilitate access to cloud resources via managed cloud services that integrate open source tools.
+- **Application guidance and training** - to provide community leaders with the skills to utilize this infrastructure for the needs. This includes language-localized content and training materials.
+- **Training for trainers** - to provide community leaders with skills to share these workflows with others in their communities.
+- **Community leadership and management** - to provide community leaders with skills to sustain and grow healthy communities of practice.
+
+Over the course of two years, the team will provide a combination of the services described above for communities in Latin America and Africa, with the goal of understanding how such services can be most useful for these communities, how we can structure them to provide community representation in the direction of these services, and how we can sustain and scale this model of community-focused services for a global community.
+
+Importantly, we wish to do this work in a way that centers the communities we work with as co-leaders and collaborators in these services.
+We will explore ways to run these services and workshops in a way that is transparent, inclusive, and gives agency to these communities.
```

Comment:
```suggestion
We will explore ways to run these services and workshops so that they are transparent, inclusive, and give agency to the communities they support.
```